projects/components/src/file-upload/file-upload-validators.ts

```diff
@@ -0,0 +1,133 @@
+import { AbstractControl } from '@angular/forms';
+import { sumBy } from 'lodash-es';
+import { FileItem } from '../file-display/file-item';
+import { UploadFileType } from './upload-file-type';
+
+/**
+ * @description
+ * Provides a set of validators specifically for file upload component that can be used by form controls.
+ */
+// tslint:disable:no-null-keyword
+// tslint:disable:no-unnecessary-class
+export class FileUploadValidators {
```

Comment:
Do we need to make this a class? We can always export it as const methods no?

Reply:
I mean yeah, I was also confused here, just wanted to use the same approach which angular does for Validators

Reply:
I think we should just make these exported validator methods. That way we won't have to disable the tslint rule

Reply:
Done